### ISSUE TITLE
alp as a variable name is not descriptive

### ISSUE THREAD

**Issue by bnord** [GeorgiadouAntigoni/weak_lensing_machine_learning#1]
Let's change "alp" so that we know what it means when looking at it 
OR
right down somewhere what the variable means.

**Comment by GeorgiadouAntigoni** [comment]
**Alp** is the constant that the # of neurons each layer is multiplied with, to look at the impact the size of each layer has on the AE performance.

**Comment by bnord** [comment]
do you want to change the name of the variable so it's more readable?

**Comment by GeorgiadouAntigoni** [comment]
Yes, I'll change the multiplication constant of the # of neurons from alp to:

multipl_neur
